src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -274,8 +334,16 @@ export class CancerSummaryChart extends React.Component<CancerSummaryChartProps,
                     return [
                         {
                             target: "data",
-                            mutation: () => {
-                                self.barPlotTooltipModel = null;
+                            mutation: async () => {
+                                // Freeze tool tip position and give user a moment to mouse over it
+                                self.shouldUpdatePosition = false;
+                                await sleep(100);
```

Comment:
you don't have to change this but i much prefer setTimeout to sleep in a case like this.  it's just a lot more clear that this bit of code is meant to execute at deferred time.